docs/user/subprojects.rst

```diff
@@ -1,46 +1,74 @@
 Subprojects: host multiple projects on a single domain
-=====================================================
+======================================================
 
-Projects can be configured in a nested manner, by configuring a project as a
-*subproject* of another project. This allows for documentation projects to share
-a search index and a namespace or custom domain, but still be maintained
-independently.
+In this article, you can learn more about how several documentation projects can be combined and presented to the reader on the same website.
+
+Read the Docs projects can be configured to make other *subprojects* available on the website of the *main project*.
+This allows for documentation projects to share a search index and a namespace or custom domain,
+but still be maintained independently.
 
 This is useful for:
 
 * Organizations that need all their projects visible in one documentation portal or landing page
 * Projects that document and release several packages or extensions
 * Organizations or projects that want to have a common search function for several sets of documentation
 
-For example, a parent project, ``foo`` is set up with a subproject, ``bar``. The
-documentation for ``foo`` will be available at:
+For example, a parent project, ``example-project`` is set up with a subproject, ``plugin``. The
```

Comment:
Should we follow our own advice and name it `example-project-plugin`, with an alias of `plugin`?